controls/V-233547.rb

```diff
@@ -59,37 +59,29 @@
   tag cci: ["CCI-001814"]
   tag nist: ["CM-5 (1)"]
 
-pg_ver = input('pg_version')
+pg_ver = input('pg_version')  #not in use
 
-pg_dba = input('pg_dba')
+pg_log_dir = input('pg_log_dir')  #not in use
 
-pg_dba_password = input('pg_dba_password')
 
-pg_db = input('pg_db')
-
-pg_host = input('pg_host')
-
-pg_log_dir = input('pg_log_dir')
-
-pg_audit_log_dir = input('pg_audit_log_dir')
-
-sql = postgres_session(pg_dba, pg_dba_password, pg_host, input('pg_port'))
+sql = postgres_session(input('pg_dba'), input('pg_dba_password'), input('pg_host'), input('pg_port'))
 
   #Execute an incorrectly-formed SQL statement with bad syntax, to prompt log ouput
-  if file(pg_audit_log_dir).exist?
+  if file(input('pg_audit_log_dir')).exist?
+
 	describe sql.query("CREATE ROLE pgauditrolefailuretest; SET ROLE pgauditrolefailuretest; SET pgaudit.role='test'; SET ROLE postgres; DROP ROLE IF EXISTS pgauditrolefailuretest;", [pg_db]) do

```

Comment:
Looks like `pd_db` needs `input()` around it since it's not a local variable anymore @damif5

Reply:
Done